### ISSUE TITLE
Create a comms plan for WCAG 2.2 updates

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#3310]
## What
Explore how we want to communicate updates to meet WCAG 2.2 (see [pull request](https://github.com/alphagov/govuk-design-system/pull/3090) and [preview pages](https://deploy-preview-3090--govuk-design-system-preview.netlify.app/accessibility/wcag-2.2/)), and agree comms activities.

[Comms plan in Google doc](https://docs.google.com/document/d/1idlkUtopSuqGMXOwSsHVH3_oAvCK7lxX1mewZopDySo/edit?usp=sharing)

## Why
So we can define
- what should be communicated
- who should receive that information
- when we should communicate
- where we should communicate

## Who needs to work on this
Kelly, Steve, Calvin, Charlotte

## Who needs to review this
Accessibility Monitoring team, DSP comms team

## Done when
- [x] We've considered about communicating changes with research / workshop participants before release
- [x] We've considered ways in which we can reach a wider audience than our usual channels 
- [x] Liaise with Gareth in P&S comms team
- [x] We understand what we want to communicate
- [x] We've planned and decided what we're saying and where
- [x] We have a graphic to be used in comms
- [x] We have reviewed the launch blog and shared with Accessibility blog owner
- [x] We have drafted all comms messages

**Comment by kellylee-gds** [comment]
Notes from retro on 28/11:

- Include Service Manual team in WCAG 2.2 comms chats
- Hold a kick off meeting for planning WCAG 2.2 workshop with GDS & CDDO colleagues [Kelly, Steve, Calvin, Charlotte]
- Look into whether W3C are planning any future comms regarding WCAG 2.2 (as part of workshop)

**Comment by kellylee-gds** [comment]
Mvp comms announcement: mailing list, then scope the other comms activities.

Needs: teams wanted to see a11y guidance more clearly rather than baked into our components and patterns, what do I need to do, how long do I have

**Comment by kellylee-gds** [comment]
Start a doc with mvp comms which will be mailing list update. Include all important information in there. Can build other comms from that.
